tools/benchmark/src/Configuration.ts

```diff
@@ -113,11 +112,9 @@ export type BenchmarkRunningOptions =
 	| BenchmarkAsyncArguments
 	| CustomBenchmarkArguments;
 
-export type BenchmarkRunningOptionsSync = BenchmarkSyncArguments & BenchmarkTimingOptions & OnBatch;
+export type BenchmarkRunningOptionsSync = BenchmarkSyncArguments;
 
-export type BenchmarkRunningOptionsAsync = BenchmarkAsyncArguments &
-	BenchmarkTimingOptions &
-	OnBatch;
+export type BenchmarkRunningOptionsAsync = BenchmarkAsyncArguments;
```

Comment:
Nit: these two feel redundant now, maybe we should remove them?

Reply:
Fine with me. @CraigMacomber ?